app/identifiers/register/establishers/partnership/IsPartnershipDormantId.scala

```diff
@@ -30,9 +32,28 @@ case class IsPartnershipDormantId(index: Int) extends TypedIdentifier[Declaratio
 object IsPartnershipDormantId extends Enumerable.Implicits {
   override def toString: String = "isPartnershipDormant"
 
-  implicit val cya: CheckYourAnswers[IsPartnershipDormantId] =
-    IsDormantCYA(
-      label = "messages__partnership__checkYourAnswers__isDormant",
-      changeIsDormant = "messages__visuallyhidden__partnership__dormant"
-    )()
+  val label: String = "messages__partnership__checkYourAnswers__isDormant"
```

Comment:
Dormant pages are bypassed in Variations journey, hence this code is not required

Reply:
reverted